extensions/aerospace/src/utils/appSwitcher.tsx

```diff
@@ -0,0 +1,60 @@
+import { environment, popToRoot } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+import { spawnSync } from "child_process";
+
+interface Window {
+  "app-name": string;
+  "window-title"?: string;
+  "window-id": number;
+  "app-pid": string;
+  workspace: string;
+  "app-bundle-id": string;
+  "app-path": string;
+}
+
+interface Windows extends Array<Window> {}
+
+export function env() {
+  return { PATH: "/usr/local/bin:/usr/bin:/bin:/opt/homebrew/bin:" };
+}
+
+function getAppPath(bundleId: string) {
+  const appPath = spawnSync("mdfind", [`kMDItemCFBundleIdentifier="${bundleId}"`], {
+    env: env(),
+    encoding: "utf8",
+    timeout: 15000,
+  });
+  console.log(appPath.stdout.trim());
+
+  return appPath.stdout.trim();
+}
+
+export function getWindows() {
+  const aerospaceArr = spawnSync("sh", [`${environment.assetsPath}/scripts/list-windows.sh`], {
```

Comment:
For performance reasons, it's better to inline `list-windows.sh` script directly to `spawnSync` arguments